packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -877,6 +877,22 @@ class _LargeTitleNavigationBarSliverDelegate
       return navBar;
     }
 
+    int _iosSpecificLargeTitleResizer = 0;
+
+    // this is added for declare ios version as int, i can get it directly, if you can please replace it
+    if (Platform.isIOS) {
+      final _iosVersion = int.tryParse(
```

Comment:
```suggestion
      final int _iosVersion = int.tryParse(
```
   info • Missing type annotation • packages/flutter/lib/src/cupertino/nav_bar.dart:884:7 • always_specify_types

Reply:
okay